ihmc-perception/src/test/java/us/ihmc/perception/cuda/CUDAFootstepOptimizerTest.java

```diff
@@ -0,0 +1,81 @@
+package us.ihmc.perception.cuda;
+
+import org.junit.jupiter.api.Test;
+import us.ihmc.euclid.referenceFrame.FramePose3D;
+import us.ihmc.sensorProcessing.heightMap.HeightMapData;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CUDAFootstepOptimizerTest
+{
+   @Test
+   public void findMinimumCost()
```

Comment:
Try changing this to a repeated test locally and run it a bunch of times. I'm finding that it fails on occasion. 
![image](https://github.com/user-attachments/assets/41ee1e3f-c2c7-4c5e-9ef2-ce4adfb8051b)

I don't want to merge an occasionally failing test since we've got ihmc-perception passing quite reliably. I'm fine if you just disable the test for now since we know footstep streaming works, but if you can fix it that's even better. 

Btw just gotta change the annotation like this
```suggestion
   @RepeatedTest(100)
   public void findMinimumCost()
```